src/sentry/static/sentry/app/views/alerts/wizard/index.tsx

```diff
@@ -55,4 +99,33 @@ const StyledPageHeader = styled(PageHeader)`
   margin-bottom: ${space(4)};
 `;
 
+const Heading = styled('h1')`
+  font-size: ${p => p.theme.fontSizeExtraLarge};
+  font-weight: normal;
+  margin-bottom: ${space(1)};
+`;
+
+const WizardBody = styled('div')`
+  display: flex;
+`;
+
+const WizardOptions = styled('div')`
+  flex: 3;
+`;
+
+const WizardPanel = styled(Panel)`
+  padding: ${space(3)};
+  flex: 5;
+  position: relative;
+`;
+
+const StyledCreateAlertButton = styled(CreateAlertButton)`
+  position: absolute;
```

Comment:
there's all this flex goodness and this???

Reply:
haha uh ohhh 😅 yup ill change this